e2e/test/scenarios/onboarding/search/search.cy.spec.js

```diff
@@ -129,6 +131,8 @@ describe("scenarios > search", () => {
         "eq",
         `/question/${ORDERS_QUESTION_ID}-orders`,
       );
+
+      cy.get("@search.all").should("have.length", 1);
```

Comment:
It's a flaky test so it's not going to mess up every time. Try and put `cy.wait(1000)` after line 119, there should be two requests:

* `GET /search?q=o`
* `GET /search?q=ord`

Sometimes this will happen if the test runner is slow but when testing on local it's something that you won't see

Reply:
This change fixed it :tada: 056d97a4c4bc91d4ed1af25799b7a2fe7a424d7d